datalad/distribution/tests/test_create_sibling.py

```diff
@@ -946,6 +946,7 @@ def test_preserve_attrs(src=None, dest=None):
         assert fp.read() == "This is test text."
 
 
+@skip_if_on_windows
```

Comment:
I think we better keep this one (with some annotation) for when we never expect that test to not be ran on windows, and use `@known_failure_windows` which would skip them by default but otherwise so at some point we could easily trigger sweep over some issues which might no longer be failures etc.

Reply:
Right, I forgot I wanted to change this. Pushed a change.